templates/repo/branch_dropdown.tmpl

```diff
@@ -70,8 +70,8 @@
 <div class="js-branch-tag-selector {{if .ContainerClasses}}{{.ContainerClasses}}{{end}}">
 	{{/* show dummy elements before Vue componment is mounted, this code must match the code in BranchTagSelector.vue */}}
 	<div class="ui dropdown custom">
-		<button class="branch-dropdown-button gt-ellipsis ui basic small compact button gt-df gt-m-0">
-			<span class="text gt-df gt-ac gt-mr-2">
+		<button class="branch-dropdown-button ui basic small compact button gt-df gt-m-0">
+			<span class="text gt-df gt-ac gt-overflow-x-auto gt-as">
```

Comment:
I would suggest to avoid adding more names. What is an `as` helper .... it's difficult to read.

Reply:
Agreed. I also think that using `!important` on every helper class is not a good practice, and that reinventing Tailwind is probably not a great idea. But it is what it is, and it's out of scope. There are existing issues that cover this topic like [this one](https://github.com/go-gitea/gitea/issues/27677#issuecomment-1768925072).
https://github.com/go-gitea/gitea/blob/816e46ee7ce4b2649479554a940ecbe1cc505a3d/web_src/css/helpers.css#L5-L16

Reply:
We can use long names. Important on these helpers will likely remain necessary to overwrite conflicting styles, primarily from Fomantic UI, unfortunately.